nl_NL/LC_MESSAGES/tickets.po

```diff
@@ -206,37 +212,37 @@ msgid "TICKET_DESC_neighborhood_watch_ticket"
 msgstr "Bedankt dat je helpt Sudomemo een veilige en gastvrije plek te houden!"
 
 msgid "TICKET_NAME_featured_flipnote_ticket"
-msgstr "Featured Flipnote Ticket"
+msgstr "Uitgelichte Flipnote Ticket"
 
 msgid "TICKET_DESC_featured_flipnote_ticket"
-msgstr "Congratulations! One of your Flipnotes was featured."
+msgstr "Gefeliciteerd! Je flipnote is uitgelicht!"
 
 msgid "TICKET_NAME_featured_artist_ticket"
-msgstr "Featured Artist Ticket"
+msgstr "Uitgelichte Artiest Ticket"
 
 msgid "TICKET_DESC_featured_artist_ticket"
-msgstr "We give our warm congratulations to you for becoming a Sudomemo Featured Artist!"
+msgstr "We geven je een warme felicitatie aan jou omdat je een uitgelichte artiest op Sudomemo bent geworden!"
```

Comment:
Your translated of "featured artist" doesn't match the ticket name. Whatever the translation is, all words should have the first letters capitalized like the original "Sudomemo Featured Artist".